### ISSUE TITLE
Trouble with Graphics 

### ISSUE THREAD

**Issue by flyre5** [Bunsan/TechNodeFirmaCraft#47]
Hi, it appears that my graphics driver is not up for running TechNodeFirmaCraft. It's an older, built-in graphics card and the driver is not supported these days. 

So, what options would you recommend I tweak to allow my computer to run this pack?
What packs are more graphics heavy? 
Or does this pack have a Lite version...?

Thanks for your time!

More about my computer:
Windows 10 Pro
64 bit operating system
8 GB RAM
Intel Core i7
Intel Graphics 3000
64 bit Java 1.8 currently installed

**Comment by Bunsan** [comment]
I'd recommend not including Better Foliage or Localized Weather. Both are graphically intensive. 

Another option is to include Optifine, which allows you to turn off a lot more of the animations, particles, etc. Results with Optifine vary. It may improve things or make them worse. You'd just have to test. 
Test without Fastcraft. For most people it helps, but it may be making things worse for you. 

What errors are you currently getting. More info may help me give better advise.

**Comment by flyre5** [comment]
Thank you.  I put a copy of the log below in a .txt file with the error messages. 

The game crashes during Initialization. I found that sometimes crashing is fixed by disabling the splash screen in the _config_ folder, but I couldn't find the _splash.properties _ file. Could you point me to disabling the splash screen?

The errors I get are about OpenEye, then "Couldn't set pixel format" and "Pixel format not accelerated" (the latter two are involved with the graphics driver I think--I would update the driver if I could), and then "No OpenGL context found in the current thread." 

Error messages at lines: 
175, 176, 194, 195, --OpenEye
199, 200, 224  --pixel format not accelerated
282, 287  --OpenGL 

[Crash log 2_13_16.txt](https://github.com/Bunsan/TechNodeFirmaCraft/files/129496/Crash.log.2_13_16.txt)

**Comment by flyre5** [comment]
I just found this: On first- and second- generation Intel HD graphics chipsets, a Java version below 8u60 is needed on Windows 10.  
So, I'm going to try using a different version of java to see that fixes it.

**Comment by Bunsan** [comment]
Yeah all things I'm finding say the issue is win 10 not playing nice with OpenGL and Java. The other possibility is you could run in comparability mode. Can't help beyond that as I don't use Windows.

**Comment by flyre5** [comment]
Good news! I had the game running just now. Thank you for your help!

Here's what I did:
I uninstalled my version of java (1.8_u73)
Installed Java 1.8_u51 (64bit) which I downloaded from Java Archives page [http://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html](url)
I ran the TechNodefirmacraft 2.13.2  without Better Foliage or Localized Weather. 
It seemed to work even though I had many error messages about missing textures. 

This page was helpful: [http://hopper.minecraft.net/help/pixel-format-not-accelerated/](url)
